seqr/views/apis/individual_api.py

```diff
@@ -681,7 +681,7 @@ def _get_record_updates(record, individual, invalid_values, allowed_assigned_ana
                 parsed_val = v
             else:
                 parsed_val = INDIVIDUAL_METADATA_FIELDS[k](v)
-                if (k not in {FEATURES_COL, ABSENT_FEATURES_COL} and parsed_val == getattr(individual, k)) or has_same_features:
+                if (k in {FEATURES_COL, ABSENT_FEATURES_COL} and parsed_val == getattr(individual, k)) or has_same_features:
```

Comment:
Do you think this should be something like this?

```py
if has_same_features and ((k in {FEATURES_COL, ABSENT_FEATURES_COL}) or parsed_val == getattr(individual, k)):
```

Reply:
One more attempt, how about:

```suggestion
                                if (
                    (has_same_features and k in {FEATURES_COL, ABSENT_FEATURES_COL})
                    or parsed_val == getattr(individual, k)
                ):
```